cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -279,6 +279,7 @@ handleSimpleNode p trace nodeTracers nc onKernel = do
     , dtHandshakeTracer = handshakeTracer nodeTracers'
     , dtHandshakeLocalTracer = localHandshakeTracer nodeTracers'
     , dtDiffusionInitializationTracer = diffusionInitializationTracer nodeTracers'
+    , dtLedgerPeersTracer = nullTracer -- TODO network team
```

Comment:
@karknu @coot: If you'd like a tracer, feel free to push a commit adding one.

Reply:
`nullTracer` is fine for now. The code that use ledger peers aren't in master.